src/renderer/components/app-core.tsx

```diff
@@ -116,8 +116,11 @@ export class CoreApplication extends React.Component<CoreAppProps, Partial<CoreA
       window.location.reload();
     }
 
-    this.addFilesToState(sortedUnzippedFiles, 'state', 'netlog');
-    this.addFilesToState(sortedUnzippedFiles, 'state', 'trace');
+    if (sortedUnzippedFiles.trace.length > 0) {
+      this.addFilesToState(sortedUnzippedFiles, 'state', 'trace');
```

Comment:
Good catch! Looking at that method I think I misunderstood how it worked. Can you try replacing this section with a single call to `this.addFilesToState(sortedUnzippedFiles, 'state', 'netlog', 'trace')`